eoscostarica/guias.eoscostarica.io
# update(frontend): Herramientas favoritas

### GH Issue

Resolves #219 

### Steps to test
1. Go to Herramientas Favoritas
2. Check the content is right


#### CheckList
- [ ] The content is equal to https://guide.eoscostarica.io/docs/developer-tools
